frontend/public/components/operator-lifecycle-manager/_operator-lifecycle-manager.scss

```
@@ -140,3 +140,22 @@
   align-items: center;
   height: 25px;
 }
+
+.co-create-operand__header {
+  border-bottom: 1px solid #ddd;
+  padding: 10px 15px;
+  @media(min-width: $screen-sm-min) {
+    padding: 10px 31px; // left alignment with namespace selector text
+  }
+}
+
+.co-create-operand__header__buttons {
```

Avoid nesting blocks in BEM. See http://getbem.com/faq/

Maybe...?

```suggestion
.co-create-operand__header-buttons {
```